docs/GettingStarted.md

```diff
@@ -1990,8 +1990,8 @@ end
 | `tracing.partial_flush.enabled`                         |                                | `false`                                                           | Enables or disables partial flushing. Partial flushing submits completed portions of a trace to the agent. Used when tracing instruments long running tasks (e.g. jobs) with many spans.                                                  |
 | `tracing.partial_flush.min_spans_threshold`             |                                | `500`                                                             | The number of spans that must be completed in a trace before partial flushing submits those completed spans.                                                                                                                              |
 | `tracing.sampler`                                       |                                | `nil`                                                             | Advanced usage only. Sets a custom `Datadog::Tracing::Sampling::Sampler` instance. If provided, the tracer will use this sampler to determine sampling behavior. See [Application-side sampling](#application-side-sampling) for details. |
-| `tracing.sampling.default_rate`                         | `DD_TRACE_SAMPLE_RATE`         | `nil`                                                             | Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%, recommended). See [Application-side sampling](#application-side-sampling) for details.                                                                                   |
-| `tracing.sampling.rate_limit`                           | `DD_TRACE_RATE_LIMIT`          | `100`                                                             | Sets a rate limit for sampling. See [Sampling](#sampling) for more details.                                                                                                                                                               |
+| `tracing.sampling.default_rate`                         | `DD_TRACE_SAMPLE_RATE`         | `nil`                                                             | Sets the trace sampling rate between `0.0` (0%) and `1.0` (100%). See [Application-side sampling](#application-side-sampling) for details.                                                                                                  |
+| `tracing.sampling.rate_limit`                           | `DD_TRACE_RATE_LIMIT`          | `100`                                                             | Sets a maximum number of traces per second to sample. Defaults to 100 per second. Set a rate limit to avoid the ingestion volume overages in the case of traffic spikes.                                                                    |
```

Comment:
Minor: I like the "per second" note -- I suggest moving it into the default column and avoid repeating it in the description 😄 

```suggestion
| `tracing.sampling.rate_limit`                           | `DD_TRACE_RATE_LIMIT`          | `100` (per second)                                                | Sets a maximum number of traces per second to sample. Set a rate limit to avoid the ingestion volume overages in the case of traffic spikes.                                                                    |
```